Dockerfile

```diff
@@ -19,10 +19,14 @@ FROM base as build
 
 # Install packages needed to build gems and node modules
 RUN apt-get update -qq && \
-    apt-get install --no-install-recommends -y build-essential curl git libpq-dev libvips node-gyp pkg-config python-is-python3
+    apt-get install --no-install-recommends -y \
+    build-essential \
+    curl \
+    libmariadb-dev  `# Needed by ActiveRecord` \
+    chromium  # Needed by grover
```

Comment:
So if I remember correctly, Grover/Puppeteer fetches its own binary for Chromium and we just need to make sure it has all of the dependencies it needs (but not Chromium itself, which depends on X window packages).

Reply:
It doesn't on aarch64. I'm also concerned about versioning issues with this. It might be safer, though less convenient, to just restrict it to x86-64.

Reply:
Yeah if you use the Chromium provided by Puppeteer then you are guaranteed that the version will match. Also _most_ of the functionality will work fine on aarch64, just not PDF generation. I think it is fine to restrict it to x86. I also definitely would not want all of the X dependencies that come with the Chromium package in the Docker image.

Reply:
Puppeteer doesn't install at all without chromium. That can be switched off with an environment flag, but then it wouldn't bring its own binary. I also don't like the idea of conditional builds nor keeping differing environments on different platforms, so I'd also like to just restrict platform.